private/bufpkg/bufimage/bufimagetesting/bufimagetesting_test.go

```diff
@@ -455,22 +455,19 @@ func TestBasic(t *testing.T) {
 	)
 	diff := cmp.Diff(protoImage, bufimage.ImageToProtoImage(newImage), protocmp.Transform())
 	require.Equal(t, "", diff)
-	// TODO: all of the below proto equality checks should use cmp.Diff
```

Comment:
I had to address this TODO because now that the `ProtoReflect` method might be called, the internal, unexported `atomicMessageState` pointer field of the message can differ between the two messages being compared.